### PR TITLE
add new recipe for  dawet

### PR DESCRIPTION
**Pull request by pratamawijaya**
New Recipe for dawet, specialty drink from Janavenese, Indonesia

### CODE REVIEW

File: src/recipes/dawet-rice-flours.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            "ingredient": "Water"
+        }
+    ],
+    "directions": [
+        "Prepare ingredients, mix all ingredients until there are no lumps of flour",
+        "Turn on the low heat, stir until the mixture begins to thicken and weigh, turn off the heat, prepare ice cubes and give enough water in a rather large container, to print the cendol mixture with cendol mold.",
+        "For the sugar sauce: boil all ingredients until the sugar dissolves stir and strain.",
+        "For the coconut milk: boil all ingredients to boil and take the leaf leaves to wait for cold, prepare and arrange cendol along with sugar sauce and coconut milk according to taste."
+    ],
+    "image": "dawet.jpg",
```
Comment (alfg) [inline src/recipes/dawet-rice-flours.json]:
```suggestion
    "image": "dawet-rice-flours.jpg",
```
Hi, can you rename the image filename to match the json filename? Thanks.

Comment (pratamawijaya) [reply src/recipes/dawet-rice-flours.json]: done

File: src/recipes/dawet-rice-flours.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Dawet is a special drink from Javanese Island, its made from rice flour and cornstarch. It serve with brown sugar sauce.",
+    "description": "Es Dawet",
```
Comment (alfg) [inline src/recipes/dawet-rice-flours.json]: One more suggestion: The drink name should just be the name. The rest can go into the description.

**Comment by alfg** [comment]
Thanks for the quick fixes!

**Pull request merged**